Documentation/ColumnsConfig/Type/Category/Index.rst

```diff
@@ -29,12 +29,13 @@ The following options can be overridden via :ref:`page TSconfig, TCE form
 *  `readOnly`
 *  `treeConfig`
 
-
 .. note::
 
    It is still possible to configure a category tree with `type=select`
    and `renderType=selectTree`. This configuration will still work, but
    it can in most cases be simplified by using the new :php:`category` TCA type.
+   However if you need to modifiy e. g. the `foreign_table_where` field, you
```

Comment:
The note sounds a little bumpy now. "replicate the category type" is about the same like "configure a category tree with `type=select`
   and `renderType=selectTree`" all you are adding is one example when that would be neccessary. To it could be added as an example in the sentence above

Reply:
I hope my updated pull request is better now.